DI-Policies.md

```diff
@@ -0,0 +1,67 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+      + Examples include, but are not limited to:
+         + Promotion of diversity-and-inclusion-minded vendors in my social feeds and blog
+         + Sharing love stories from underrepresented folx in my social feeds and blog
+
+   + Use my privilege, and speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website:
+      + Ensure representation of underrepresented groups in imagery across website, including blog
+      + Ensure the use of inclusive language (e.g. gender neutral terms) across website, including blog
+
+   + Monthly for social:
+      + Ensure representation of underrepresented groups in feeds, across all platforms (Instagram, Facebook, Twitter, Pinterest)
+      + Ensure the use of inclusive language extends to all social posts
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+      + Questions partners will be expected to answer include:
+         + What is your understanding and experience of Diversity and Inclusion, whether in the wedding industry or outside of it?
+         + What are the D+I (Diversity and Inclusion) policies for your own business? Do you have them written down? Can you share them with me?
```

Comment:
This comment isn't about content, but presentation. I believe it might read a bit better if you add `(D+I)` to line [38](https://github.com/nadiajoyce/NJP-Docs/pull/2/files#diff-efbf3c831ea5220bfe2383c12baf5211R38) and stick to using the abbreviation from then on.

Reply:
Thank you! I knew I was probably making some mistakes like this in terms of format, and I appreciate your eyes on that. Will definitely update!